modules/mongodb/mount/entrypoint.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
```

Comment:
I'd rename this to `entrypoint-tc.sh` in order to make it explicit this is added and executed by Testcontainers. Also adding a comment with something like "# Injected by Tescontainers" could be interesting.